apps/browser/src/scripts.ts

```diff
@@ -1,4 +1,40 @@
-export const onGoalAchievedScript = {
+import { InMemoryWorkspace, Workspace } from "@evo-ninja/agent-utils";
+
+export function createInBrowserScripts(): InMemoryWorkspace {
+  const workspace = new InMemoryWorkspace();
+
+  const availableScripts = [
```

Comment:
I was imagining this being auto generated so it can inject any scripts that exist within the `./scripts` directory.